scope/src/doctor/commands/run.rs

```diff
@@ -33,6 +33,9 @@ pub struct DoctorRunArgs {
     /// Do not ask, create report on failure
     #[arg(long, default_value = "false", env = "SCOPE_DOCTOR_AUTO_PUBLISH")]
     pub auto_publish_report: bool,
+    /// Automatically approve all fix prompts without asking
+    #[arg(long, default_value = "false")]
+    pub yolo: bool,
```

Comment:
I actually think we should also have a short flag (`-y`) for this as well.
`-y` is typically used for this in other applications, but I do appreciate the long flag being `--yolo`.